input/resources/OperationDefinition-deqm.evaluate-measure.xml

```diff
@@ -91,11 +91,11 @@
         <searchType value="reference"/>
     </parameter>
     <parameter>
-        <name value="practitioner"/>
+        <name value="provider"/>
         <use value="in"/>
         <min value="0"/>
         <max value="1"/>
-        <documentation value="Practitioner (references [QICore Practitioner](http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-practitioner)) for which the measure will be calculated. If specified, the measure will be calculated only for subjects that have a primary relationship to the identified practitioner"/>
+        <documentation value="The provider for which the report will be run. This may be a reference to a Practitioner, PractitionerRole, or Organization. If specified, the measure will be calculated for subjects that have a primary relationship to the identified provider. How this relationship is determined is implementation-specific. If specified, the measure will be calculated only for subjects that have a primary relationship to the identified provider"/>
```

Comment:
Remove the last sentence "If specified, the measure will be calculated only for subjects that have a primary relationship to the identified provider". It is redundant.

Reply:
removed sentence as requested